Xamarin.Forms.Platform.iOS/Renderers/ImageElementManager.cs

```diff
@@ -80,6 +80,27 @@ public static void SetAspect(IImageVisualElementRenderer renderer, IImageElement
 			Control.ContentMode = imageElement.Aspect.ToUIViewContentMode();
 #else
 			Control.Layer.ContentsGravity = imageElement.Aspect.ToNSViewContentMode();
+
+			switch (imageElement.Aspect)
+			{
+
+				case Aspect.AspectFill:
+					// No built in aspect fill on the NSImageView directly, it is there on the layer so go through there
+					if (Control.Image != null)
+					{
+						Control.Layer = new CALayer();
```

Comment:
We should check if we have a Layer before creating one, the ViesualElementTracker set s layer properties.

if( Control.Layer != null)
    Control.Layer = new CALayer();

Reply:
Do you mean `if( Control.Layer != null)` like you said or `if( Control.Layer == null)` to just create a Layer once?

Reply:
right if null :P 

getting back from vacations :)

Reply:
So, the weird thing is when I implement that it stops working... 🤔

Reply:
Ok so something wrong with this fix :P you need to investigate the work it's done on the tracker and this i think.

Reply:
Yep, I'll mark it DNM for now and revisit later.

Reply:
Could be something related to when `wantsLayer = true` happens relative to a default backing layer getting created prior to this point? It is apparently [sensitive to order](https://stackoverflow.com/a/38752428/1735721).